static/coffee/flows/controllers.coffee

```diff
@@ -1640,6 +1650,7 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
         ruleset.config =
           webhook: formData.webhook
           webhook_action: formData.webhook_action
+          webhook_header: formData.webhook_header
```

Comment:
This can be more than one header ya? So should rename this to `webhook_headers` everywhere.